StructuralPreprocessing/SubmitStructuralPreprocessingBatchHPC

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
As with others, let's remove this from this pull request and put it in a separate feature.